### ISSUE TITLE
Time and Position Plot nodes cannot be moved using keyboard navigation

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/quantum-wave-interference#185]
**Test device**
Chromebook

**Operating System**
ChromeOS

**Browser**
Chrome

**Problem description**
From https://github.com/phetsims/qa/issues/1405

When navigating with the keyboard, there is no apparent way to move the Time Plot and Position Plot nodes. The nodes can be repositioned using the mouse, but equivalent keyboard controls do not appear to be available.

**Steps to reproduce**
1.  In the High Intensity or in the Single Particles screen, check the Time Plot and Position Plot checkboxes.
2. Press tab to focus these items on the screen. You will notice that keyboard controls do not appear to be available for them.

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Quantum Wave Interference‬
URL: https://phet-dev.colorado.edu/html/quantum-wave-interference/1.0.0-dev.3/phet/quantum-wave-interference_all_phet.html
Version: 1.0.0-dev.3 2026-06-08 17:20:20 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/148.0.0.0 Safari/537.36
Language: en-US
Window: 1600x812
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by samreid** [comment]
OK I added a rough draft of alt-input and core description to both, @vperezcadavid99 can you please test and advise?

**Comment by vperezcadavid99** [comment]
Thanks @samreid it looks good in main!
Currently, the interactive highlight only covers the gray portion of the Position Plot, whereas the Time Plot highlight encompasses the entire plot area. I wanted to confirm if this is the correct way to show the interactive highlight.

<img width="518" height="205" alt="Image" src="https://github.com/user-attachments/assets/6dbae027-a536-4c08-9463-b4b5ca9edc8e" />

**Comment by matthew-blackman** [comment]
The Position Plot’s interaction bounds now encompass both the aperture probe and the graph panel.

The combined rectangular bounds are used consistently for mouse/touch input, keyboard focus highlighting, and interactive highlighting. The full area, including the gap between the aperture and graph, can now be dragged as one tool.

@samreid and @vperezcadavid99, can you please review?

**Comment by samreid** [comment]
Looks good in my review and testing, @vperezcadavid99 can you please test and close if ok?

**Comment by vperezcadavid99** [comment]
Thanks @samreid and @matthew-blackman, it looks better now.
Closing!
